src/scss/_search.scss

```diff
@@ -6,18 +8,29 @@
     padding: .75rem !important;
     padding-left: 2.5rem !important;
     border-radius: .25rem !important;
-    border-color: #cbd5e0 !important; /* border-gray-400 */
+    border-color: map-get($gray, 400) !important; /* border-gray-400 */
     font-family: "Inter" !important;
     background: none !important;
 
     @include transition;
 
-    &:focus {
-        border-color: #52a6da !important; /* border-blue-500 */
+    &:focus:not(.registry) {
+        border-color: map-get($blue, 500) !important; /* border-blue-500 */
+    }
+
+    &.registry {
+        border-top-right-radius: 0 !important;
+        border-bottom-right-radius: 0 !important;
+
+        &:focus {
+            border-color: map-get($fuchsia, 500) !important;
```

Comment:
Would love to come back and do something to avoid having to use so many `!important`s, but I agree that it's not worth figuring out in this PR/anytime soon, since it continues the pattern already in this file + it works.

Reply:
Yeah so I think it's specific to the Swiftype widget. Without the `!important` keyword our styles seem to get overridden by whatever Swiftype's doing after it loads. I have to admit that I didn't check if there was a different way that would not require us to use all of these `!important`